scapy/utils.py

```diff
@@ -827,8 +827,10 @@ def read_packet(self, size=MTU):
             return None
         sec,usec,caplen,wirelen = struct.unpack(self.endian+"IIII", hdr)
         s = self.f.read(caplen)[:size]
-        return s, (sec, usec, caplen, wirelen)
+        return s, self._PacketInfo(sec=sec, usec=usec, wirelen=wirelen, caplen=caplen)
 
+    _PacketInfo = collections.namedtuple("PacketInfo",
```

Comment:
Please move this declaration **before** its first use, for readability. Also, since this type will be used for values returned by a public method, I don't think it should be private. Could you change this name to `PacketInfo` or even better `PacketMetadata`?

Reply:
`PacketMetadata` is confusing: we already have `Packet_metaclass`
`PacketInfo` or anything similar is good to me

Reply:
I do not agree. Metaclass and Metadata have different meanings. Metadata seems more precise (and clear) than Info to me.

Reply:
Fair enough 👍

Reply:
I'v done a changes. 
Is there a way to update this PR or I should create next one?
PS: I will deal with RawPcapNgReader, but in the next issue/PR.